src/Unleash/Internal/CachedFilesLoader.cs

```diff
@@ -49,6 +51,7 @@ public CachedFilesResult EnsureExistsAndLoad()
             if (result.InitialToggleCollection == null)
             {
                 result.InitialETag = string.Empty;
+                result.InitialToggleCollection = toggleBootstrapHandler.Read();
```

Comment:
I know this aligns with the Java SDK. But we actually think it may be a mistake to not allow it to override the locally backup file. The default should be that bootstrap can override a local cache, but it should be an option the user can set. 

`boostrapOverride = true` should default to true, and means that bootstrap will override InitialToggleCollection, given that the bootstrap collection is valid and not an empty collection.

Reply:
Ok good stuff, that makes sense! i'll add some more tests and implement this!